pandas/_libs/src/ujson/lib/ultrajsonenc.c

```diff
@@ -981,13 +996,17 @@ void encode(JSOBJ obj, JSONObjectEncoder *enc, const char *name,
             enc->iterBegin(obj, &tc);
 
             Buffer_AppendCharUnchecked(enc, '{');
+	    Buffer_AppendIndentNewlineUnchecked (enc);
+	    Buffer_AppendIndentUnchecked (enc, enc->level + 2);
```

Comment:
As mentioned this `+ 2` isn't permanent just a hack during WIP to get this to work. Something seems off with the level struct member I just haven't figured out yet